test/test_validation.py

```diff
@@ -151,5 +152,14 @@ def test_xmlrpc(self):
 
         _.buildSphinx(dataset, doc_dir, doctree_dir, config)
 
+    def test_nonjsonresponse(self):
+        config = dict(self.config)
+        config['confluence_server_url'] = 'https://example.com/'
```

Comment:
(food-for-thought only) I ponder if `example.com` is a best solution here. I believe I understand why it was chosen -- a standard/simple site which would succeed in a GET request but not have valid JSON to parse. My dislike would be that having this makes an unnecessary external requests for a tester, where this could also be achieved with a simple local socket instead (and not rely on connectivity). But -- the other validation tests need some level of connectivity (especially Cloud-based targets) and this is a unique manual test; so what is the harm in keeping it simple. I believe this URL is fine, just sharing some comments on it.

Reply:
I agree. Ideally this would use a dummy local server, but I did not feel the need as there is no other test that will use it.
Would be good to change someday.